lib/voodoo_sms.rb

```diff
@@ -46,6 +46,12 @@ def get_sms(from, to, keyword = '')
     end
   end
 
+  def get_dlr_status(reference_id)
```

Comment:
Yep, we reflect the methods from the documentation because they are sensible :stuck_out_tongue:, whereas `get_dlr_status` means more about TfL's DLR service to me!